content/en/observability_pipelines/destinations/_index.md

```diff
@@ -11,30 +11,16 @@ further_reading:
 
 Use the Observability Pipelines Worker to send your processed logs to different destinations.
 
-Select and set up your destinations when you [set up a pipeline][2]. This is step 4 in the pipeline setup process:
+Select and set up your destinations when you [set up a pipeline][1]. This is step 4 in the pipeline setup process:
 
-1. Navigate to [Observability Pipelines][1].
+1. Navigate to [Observability Pipelines][2].
 1. Select a template.
 1. Select and set up your source.
 1. Select and set up your destinations.
 1. Set up your processors.
 1. Install the Observability Pipelines Worker.
 
-{{< whatsnext desc="The available Observability Pipelines destinations are:" >}}
-    {{< nextlink href="observability_pipelines/destinations/#amazon-opensearch" >}}Amazon OpenSearch{{< /nextlink >}}
-    {{< nextlink href="observability_pipelines/destinations/amazon_s3" >}}Amazon S3{{< /nextlink >}}
-    {{< nextlink href="observability_pipelines/destinations/azure_storage" >}}Azure Storage{{< /nextlink >}}
-    {{< nextlink href="observability_pipelines/destinations/#datadog-log-management" >}}Datadog Log Management{{< /nextlink >}}
-    {{< nextlink href="observability_pipelines/destinations/#elasticsearch" >}}Elasticsearch{{< /nextlink >}}
-    {{< nextlink href="observability_pipelines/destinations/#google-chronicle" >}}Google Chronicle{{< /nextlink >}}
-    {{< nextlink href="observability_pipelines/destinations/google_cloud_storage" >}}Google Cloud Storage{{< /nextlink >}}
-    {{< nextlink href="observability_pipelines/destinations/#opensearch" >}}OpenSearch{{< /nextlink >}}
-    {{< nextlink href="observability_pipelines/destinations/#rsyslog-or-syslog-ng" >}}rsyslog or syslog-ng{{< /nextlink >}}
-    {{< nextlink href="observability_pipelines/destinations/#splunk-http-event-collector-hec" >}}Splunk HTTP Event Collector (HEC){{< /nextlink >}}
-    {{< nextlink href="observability_pipelines/destinations/#sumo-logic-hosted-collector" >}}Sumo Logic Hosted Collector{{< /nextlink >}}
-{{< /whatsnext >}}
```

Comment:
I'm curious why you got rid of this rather than changing the links to be the new link targets.

Reply:
@neko-dd Thanks for reviewing this! Because I also have the list in the left nav so I thought it was a bit redundant. But would be interested in hearing your thoughts on it because I was waffling between removing it and keeping it, since sometimes I think easier for a reader to just scroll down the page and see the list.

Reply:
Also, chatted with the PM about this. We're going to add it back in since it's easier when a reader is scrolling through the doc. :)